nautobot/extras/models/customfields.py

```diff
@@ -262,6 +298,11 @@ class CustomField(BaseModel, ChangeLoggedModel):
         "For example, <code>^[A-Z]{3}$</code> will limit values to exactly three uppercase letters. Regular "
         "expression on select and multi-select will be applied at <code>Custom Field Choices</code> definition.",
     )
+    advanced_ui = models.BooleanField(
+        default=False,
+        verbose_name="Hide from main detail tab",
+        help_text="Hide this custom field from the main detail tab. It will appear in the advanced tab instead.",
```

Comment:
Perhaps?

```suggestion
        help_text='Hide this field from the object's primary information tab. It will appear in the "Advanced" tab instead.',
```

Similar suggestion on the others.